content/applications/finance/accounting/taxation/taxes/carryover.rst

```diff
@@ -0,0 +1,58 @@
+===============
+Taxes carryover
+===============
+
+This functionality allows carrying amounts from one period to another without creating new entries.
+It responds to a specific need: under certain legal conditions, some tax report lines must have
+their amount transferred to the following period without creating additional accounting entries.
+
+.. example::
+   In the Belgian tax report, grid 81 may contain a negative amount at the end of the tax period,
+   but it must be declared to the government as zero, and the negative amount should be carried over
+   to the next period.
+
+Configuration
+=============
```

Comment:
I don't understand in which cases someone would need to configure this in the hidden Config/tax reports.
cc @oco-odoo ?

We should explain the default use first, and - unless I'm mistaken - it is all pre-configured correctly according to your localization's needs?

Reply:
Indeed, normally you won't have to configure it yourself as a customer. I guess the interest of a doc about it is more for whose localization does not natively support carryover in Odoo. For partners nedding it for their customers, for example. Maybe even to get an idea on how to configure it before making a PR for our Community repo. 

In our current code base, carryover is only supported in Belgium, France and Italy. I guess making it clearer for everyone might help having the other cases covered :)

Though, I don't know if it's the best time to do this, as we're currently entirely revamping Odoo's reports for v16, and the carryover is configured entirely differently in the new model (it's overall clearer than before).

Reply:
Thanks for your feedback @oco-odoo, so in the intro, we could clarify that per default carryover is supported for companies located in BE, FR, IT and that this doc is there to help for customers located in another location who might need this feature too?
As the doc is almost done, let's publish it for V15, what do you think @jcs-odoo ?